zerver/lib/validator.py

```diff
@@ -172,3 +173,43 @@ def check_url(var_name: str, val: object) -> Optional[str]:
         return None
     except ValidationError as err:
         return _('%s is not a URL') % (var_name,)
+
+def validate_field_data(field_data: str) -> Optional[str]:
+    """
+    This function is used to validate the data sent to the server while
+    creating/editor choices of the choice field in Organization settings.
+    """
+    if field_data == '':
+        return None
+
+    error = _("The field 'field_data' is not in proper format.")
```

Comment:
This error string is a little awkward; something like "Invalid 'field_data' format" would be better.